packages/editor/src/components/browser/CmsBrowser.tsx

```diff
@@ -63,26 +79,44 @@ const CmsBrowser = ({ value, onChange }: { value: string; onChange: (value: stri
   const [expanded, setExpanded] = useState<ExpandedState>(false as ExpandedState);
   const [globalFilter, setGlobalFilter] = useState('');
   const [rowSelection, setRowSelection] = useState<RowSelectionState>({});
+  const [columnVisibility, setColumnVisibility] = useState<VisibilityState>({ type: false, values: false });
+  const [columnFilters, setColumnFilters] = useState<ColumnFiltersState>(
+    typFilter === 'NONE' || typFilter === undefined ? [] : [{ id: 'type', value: typFilter }]
+  );
+
+  const addIvyPathToValue = (value: string, type: ContentObjectType, noApiCall?: boolean) => {
+    if (value.length === 0) {
+      return '';
+    }
+    if (type === 'FILE') {
+      return noApiCall ? value : `ivy.cms.cr("${value}")`;
+    }
+    return noApiCall ? value : `ivy.cms.co("${value}")`;
+  };
```

Comment:
instead of check this noApiCall twice, make one check at the beginning:

```ts
const addIvyPathToValue = (value: string, type: ContentObjectType, noApiCall?: boolean) => {
  if (noApiCall || value.length === 0) {
    return value
  }
  if (type === 'FILE') {
    return `ivy.cms.cr("${value}")`;
  }
  return `ivy.cms.co("${value}")`;
};
```